R/correlation.R

```diff
@@ -17,9 +21,20 @@
 #' @examples
 #' WoJ %>% correlate(ethics_1, ethics_2, ethics_3)
 #' WoJ %>% correlate()
+#' WoJ %>% correlate(ethics_1, ethics_2, ethics_3, partial = TRUE)

```

Comment:
Maybe a slightly more realistic examlpe would be helpful. For example:

`WoJ %>% correlate(autonomy_selection, autonomy_emphasis, work_experience, partial = TRUE)`